__tests__/plugin.test.ts

```diff
@@ -19,18 +21,29 @@ const THEME_CONFIG = {
 }
 
 expect.extend({
-    toMatchCss: (received, argument) => {
-        let stripped = (str: string) => str.replace(/[;\s]/g, '')
+    toMatchCss: (receivedCss, expectedCss) => {
+        let strip = (str: string) => str.replace(/[;\s]/g, '')
 
-        if (stripped(received) === stripped(argument)) {
+        if (strip(receivedCss) === strip(expectedCss)) {
             return {
                 message: () =>
-                    `expected ${received} not to match CSS ${argument}`,
+                    `expected ${receivedCss} not to match CSS ${expectedCss}`,
                 pass: true,
             }
         } else {
+            let receivedCssFormatted = prettier.format(receivedCss, {
+                parser: 'css',
+            })
+            let expectedCssFormatted = prettier.format(expectedCss, {
+                parser: 'css',
+            })
+            let diff = diffStringsUnified(
+                receivedCssFormatted,
+                expectedCssFormatted,
+            )
+            let message = `expected css to match:\n${diff}`
             return {
-                message: () => `expected ${received} to match CSS ${argument}`,
+                message: () => message,
```

Comment:
Small nit: "css" should be uppercased, and maybe the message could just be returned directly as in the passing case.